blazor/chart/how-to/synchronized-chart.md

```diff
@@ -0,0 +1,412 @@
+---
+layout: post
+title: Synchronized Chart in Blazor Charts Component | Syncfusion
+description: Checkout and learn here all about Synchronized Chart in Syncfusion Blazor Charts component and much more details.
+platform: Blazor
+control: Chart
+documentation: ug
+---
+
+<!-- markdownlint-disable MD036 -->
+
+# Synchronized chart in Blazor Charts Component
+
+Synchronized chart has the ability to access and interact with two or more charts concurrently. This means that actions performed on one chart, such as placing a cursor, clicking on a specific point, or activating a tooltip are reflect simultaneously on corresponding positions or data points across other synchronized charts based on its X-Y coordinates. 
+
+## Tooltip synchronization
+
+The tooltip can be synchronized across multiple charts using the `ShowTooltip` and `HideTooltip` methods. When we hover over a data point in one chart, we call the `ShowTooltip` method for the other charts to display related information in other connected charts simultaneously.
+
+In the showTooltip method, specify the following parameters programmatically to enable tooltip for a particular chart:
+
+* x - x-value of point or x-coordinate value.
+* y - y-value of point or y-coordinate value.
+
+```cshtml
+
+@using Syncfusion.Blazor.Charts;
+@using Syncfusion.Blazor
+@using System.Net.Http.Json
+@inject NavigationManager NavigationManager
+@inject HttpClient Http
+
+<div class="row">
```

Comment:
In the tooltip topics, the code snippet contains codes related to zooming and crosshair. Could you please clean it up?